tests/nemo_text_processing/de/data_text_normalization/test_cases_electronic.txt

```diff
@@ -1,14 +1,14 @@
-c d f at a b c punkt e d u~cdf@abc.edu
+c d f at a b c punkt edu~cdf@abc.edu
 a b c at g mail punkt a b c~abc@gmail.abc
 a b c at a b c punkt com~abc@abc.com
 a s d f eins zwei drei at a b c punkt com~asdf123@abc.com
 a eins b zwei at a b c punkt com~a1b2@abc.com
 a b drei bindestrich s d d bindestrich drei at g mail punkt com~ab3-sdd-3@gmail.com
 h t t p s doppelpunkt slash slash w w w punkt a b c punkt com~https://www.abc.com
 w w w punkt a b c punkt com~www.abc.com
-b r e t t s p i e l v e r s a n d punkt de .~brettspielversand.de.
-w w w punkt e n v e e d y a punkt net .~www.enveedya.net.
-w w w punkt a m a z o n punkt com punkt de .~www.amazon.com.de.
+brettspielversand punkt de.~brettspielversand.de.
+w w w punkt e n v e e d y a punkt net.~www.enveedya.net.
+w w w punkt amazon punkt com punkt de.~www.amazon.com.de.
```

Comment:
extra period intended?

Reply:
I'm not sure I'm following. The periods agree on both sides of the tilde. Unless I'm missing something...

Reply:
I'm just san checking you intended to include the full stop in these examples. It make the electronic test cases become a partial punctuation test case.

Reply:
Nvm i'm seeing it in your draft notes.